metadata.yaml

```diff
@@ -26,3 +26,5 @@ requires:
     limit: 1
   ingress:
     interface: ingress
+  mongodb_client:
+    interface: mongodb_client
```

Comment:
Should `mongodb_client` relation has a limit of 1 like `influxdb` or `pg`?

Reply:
That's a good question, and yes, I think it should, as it'll make the code simpler for now (or at least mean we don't need to add complexity to support multiple relations. I'll update, thanks.